depthai_sdk/src/depthai_sdk/managers/encoding_manager.py

```diff
@@ -79,8 +89,13 @@ def printManual():
         for queue in self._encodingQueues.values():
             queue.close()
 
+        for name, file in self._mcap.items():
```

Comment:
DepthAiMcap class should have a `close` method that will allow to close everything - then, it could be used inside `self._encodingFiles` parameter and closed the same way